servers/labscript/src/labscript_server/main.py

```
@@ -0,0 +1,188 @@
+        )
+
+        tmp_wrapper = tempfile.NamedTemporaryFile(
+            suffix=".py", mode="w", delete=False, encoding="utf-8"
+        )
+        tmp_wrapper.write(wrapper_code)
+        tmp_wrapper.flush()
+        tmp_wrapper.close()
+
+        # Step 3: execute
+        try:
+            result = subprocess.run(
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)